src/DisqusInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/*
+* This file is part of the KnpDisqusBundle package.
+*
+* (c) KnpLabs <hello@knplabs.com>
+*
+* For the full copyright and license information, please view the LICENSE
+* file that was distributed with this source code.
+*/
+
+namespace Knp\Bundle\DisqusBundle;
+
+interface DisqusInterface
+{
+    public function fetch(string $shortname, array $options, string $fetch = 'threads/listPosts'): array;
```

Comment:
We need to document this method.

Because the return is an array, we need to have some docs describing this - even if it is just a link to the Disqus docs that say "this will be the array representation of this documented JSON response from Disqus".